src/helpers/user.js

```diff
@@ -22,6 +23,7 @@ export function fromVMtoM(user) {
 export function fromMtoVM(user) {
   return {
     ...user,
+    ...overwriteProfileDefaults(user),
```

Comment:
Maybe we can do it only for users who are not mentors yet to not bother mentors that already use auth avatar. WDYT?

Reply:
Absolutely, that makes total sense. I will update the PR.